lib/eclipse/Deck/DeckKeyword.cpp

```diff
@@ -17,9 +17,9 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "DeckKeyword.hpp"
```

Comment:
Since this results in almost all source files of the module being touched, don't you think the directory reorganization and such stylistic non-buildsystem changes merit a separate PR? (or is this due to the PR's WIP status?)